### PR TITLE
added properties to inputbox and form to be able to change style fro…

### PR DESCRIPTION
**Pull request by llali**
added properties to input box and form to be able to change style from extension

### CODE REVIEW

File: src/sql/parts/modelComponents/formContainer.component.ts [hunk 1]
```diff
@@ -113,6 +116,16 @@ export default class FormContainer extends ContainerBase<FormItemLayout> impleme
...
 	}
 
+	private getFormWidth(item: FormItem): string {
+		let itemConfig = item.config;
+		return itemConfig && itemConfig.width ? +itemConfig.width + 'px' : '400px';
+	}
+
+	private getComponentWidth(item: FormItem): string {
+		let itemConfig = item.config;
+		return itemConfig ? +itemConfig.componentWidth + 'px' : '';
```
Comment (kburtram) [inline src/sql/parts/modelComponents/formContainer.component.ts]: what's the purpose of the `+` in from of `itemConfig`?

Comment (llali) [reply src/sql/parts/modelComponents/formContainer.component.ts]: converting number to string

---
In reply to: [186891249](https://github.com/Microsoft/sqlopsstudio/pull/1371#discussion_r186891249) [](ancestors = 186891249)

Comment (kburtram) [reply src/sql/parts/modelComponents/formContainer.component.ts]: shouldn't the `+ 'px'` force the string conversion?  or is there some case I'm missing?

Comment (llali) [reply src/sql/parts/modelComponents/formContainer.component.ts]: it does, I'll fix it

---
In reply to: [186892512](https://github.com/Microsoft/sqlopsstudio/pull/1371#discussion_r186892512) [](ancestors = 186892512)

File: src/sql/sqlops.proposed.d.ts [hunk 2]
```diff
@@ -303,7 +309,7 @@ declare module 'sqlops' {
 		initializeModel<T extends Component>(root: T): Thenable<void>;
 	}
 
-	export namespace dashboard {
+	export namespace modelView {
```
Comment (kburtram) [inline src/sql/sqlops.proposed.d.ts]: I don't know if `modelView` is the most intuitive name from this namespace.  Maybe something like `uiBuilder` or something else that is more clear of what is the purpose of this set of classes.

**Pull request merged**